python/download_tile.py

```diff
@@ -0,0 +1,77 @@
+from looker_sdk import client, models
+import time
+import sys
+
+sdk = client.setup("../looker.ini")
+
+def get_dashboard(title: str):
+  """Get a dashboard by title"""
+  dashboard = next(iter(sdk.search_dashboards(title=title)), None)
+  if not dashboard:
+    print(f"dashboard {title} was not found")
+  return dashboard
+
+def get_tile(dash: models.Dashboard, title: str):
+  """Find a dashboard tile by title"""
+  title = title.lower()
+  found = None
+  assert dash.dashboard_elements
+  for tile in dash.dashboard_elements:
+    assert tile.title
+    if tile.title.lower() == title:
+      found = tile
+  if not tile:
```

Comment:
```suggestion
  if not found:
```

Reply:
python has for/else so we could just do

```suggestion
 else:
```

and then remove the `found` name on L19 and L22 (and yes break on L22)